tools/bmh_test/run_local_bmh_test_setup.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -eux
+
+REPO_ROOT=$(realpath "$(dirname "${BASH_SOURCE[0]}")/../..")
+
+cd "${REPO_ROOT}" || exit 1
+
+# Check if the script is run as root
+if [[ $EUID != 0 ]]; then
+    echo "ERROR: This script must be run as root" 
+    exit 1
+fi
+
+# Install virtinst, libvirt clients and daemon system
+apt-get -y update
+apt-get -y install virtinst libvirt-clients libvirt-daemon-system
```

Comment:
Have we decided to only support Tilt in Ubuntu/Debian? If so, could you update the README with that information?